.travis.yml

```diff
@@ -33,8 +33,7 @@ addons:
 script: ./travis.sh 
 
 notifications:
-  email:
-    - cukes-devs@googlegroups.com
+  email: false
```

Comment:
Why not just remove this section from the config file and use the defaults instead?

[The default build notifications](https://docs.travis-ci.com/user/notifications#Notifications):
>  default, email notifications are sent to the committer and the commit author, if they are members of the repository (that is, they have push or admin permissions for public repositories, or if they have pull, push or admin permissions for private repositories).
> 
> Emails are sent when, on the given branch:
> 
>   * a build was just broken or still is broken
>   * a previously broken build was just fixed

With the change to `false` you're completely disabling notifications. I find them rather useful the way they're set by default.